securedrop/Dockerfile

```diff
@@ -29,6 +31,8 @@ COPY requirements requirements
 RUN pip install -r requirements/securedrop-app-code-requirements.txt && \
     pip install -r requirements/test-requirements.txt
 
+RUN if test $USER_NAME != root ; then useradd --no-create-home --home-dir /tmp --uid $USER_ID $USER_NAME && echo "$USER_NAME ALL=(ALL) NOPASSWD:ALL" >> /etc/sudoers ; fi
```

Comment:
You need to export the `ARG` value with `ENV`.

https://docs.docker.com/engine/reference/builder/#using-arg-variables

Reply:
done